### PR TITLE
Add com.ugetdm.uGet (uget-flatpak)

### PR DESCRIPTION
**Pull request by scx**
**uGet** is a lightweight yet powerful Open Source download manager for GNU/Linux developed with GTK+.

See also:
https://ugetdm.com
https://sourceforge.net/projects/urlget
https://github.com/scx/uget-flatpak
https://github.com/scx/flathub/tree/com.ugetdm.uGet

### CODE REVIEW

**Comment by scx** [comment]
Developers have been notified, but not answered yet.

**Comment by scx** [comment]
testbot, build com.ugetdm.uGet

**Comment by barthalion** [comment]
> Developers have been notified, but not answered yet.

Post a link to the discussion please.

**Comment by scx** [comment]
@barthalion 

> Post a link to the discussion please.

I sent an e-mail to the upstream developer regarding this.
As you can see, this project is developed on SourceForge.

File: com.ugetdm.uGet.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+app-id: com.ugetdm.uGet
+runtime: org.gnome.Platform
+runtime-version: '3.30'
```
Comment (LeandroStanger) [inline com.ugetdm.uGet.yaml]: Update runtime to v3.32

**Comment by LeandroStanger** [comment]
bot, build com.ugetdm.uGet

**Pull request closed without merging**